pipelines/edx-sysadmin-publish.yml

```diff
@@ -0,0 +1,56 @@
+---
+resource_types:
+- name: pypi
+  type: docker-image
+  source:
+    repository: cfplatformeng/concourse-pypi-resource
+    tag: latest
+
+resources:
+    - name: edx-sysadmin-repo
+      type: git
+      source:
+          uri: https://github.com/mitodl/edx-sysadmin.git
+          branch: ((release))
+
+    - name: edx-sysadmin-pypi
+      type: pypi
+      source:
+        name: edx-sysadmin
+        packaging: any
+        repository:
+          username: ((username))
+          password: ((password))
+
+
+jobs:
+- name: publish-edx-sysadmin-to-pypi
+  public: true
+  plan:
+  - get: edx-sysadmin
+    trigger: true
+
+  - task: build-and-upload
+      config:
+        platform: linux
+        image_resource:
+          type: registry-image
+          source:
+            repository: node
```

Comment:
Is the use of the node image due to a requirement for compiling JS/CSS during the package building?